railties/lib/rails/tasks/framework.rake

```diff
@@ -45,13 +45,30 @@ namespace :rails do
         @app_generator ||= begin
           require 'rails/generators'
           require 'rails/generators/rails/app/app_generator'
-          gen = Rails::Generators::AppGenerator.new ["rails"], { with_dispatchers: true },
-            destination_root: Rails.root
+
+          gen = Rails::Generators::AppGenerator.new ["rails"],
+                                                    generator_options,
+                                                    destination_root: Rails.root
+
           File.exist?(Rails.root.join("config", "application.rb")) ?
             gen.send(:app_const) : gen.send(:valid_const?)
           gen
         end
       end
+
+      private
+
+        def self.generator_options
+          options = if File.exist?(".rails")
+                      YAML.load File.read(".rails")
+                    else
+                      {}
+                    end
+          options.merge!(with_dispatchers: true)
+          options.merge!(force: true) if ENV['FORCE_RAILS_UPDATE']
+
```

Comment:
Had to add this constant so that tests don't wait for user input for updating `secrets.yml` in tests.